### DISCUSSION TITLE
Official Klaviyo integration

### DISCUSSION THREAD

**Discussion by Xytronix** [medusajs/medusa#9805] [Feature Requests]
I would love to use Klaviyo for e-mail campaigns in forms, for abandoned shopping cart and tracking customers.

Klaviyo is the tool ecom companies rely on, therefore would appreciate an official module.

**Comment by Xytronix** [comment]
I could see the option to automatically sync a customer group

**Comment by VelestonRato** [comment]
Hi @Xytronix, we at Intuio Software Labs have been working on a plugin for Klaviyo subscriptions.
The current version of this plugin is supported for Medusa version 1 only. 

https://www.npmjs.com/package/medusa-plugin-klaviyo

We are actively working on support for version 2. Please stay tuned for updates in the upcoming release.

**Reply by VelestonRato** [reply]
Support for v2 has been added to the package at

https://www.npmjs.com/package/@intuio/medusa-plugin-klaviyo

for v1 :  npm i @intuio/medusa-plugin-klaviyo@medusa-v1
for v2 : npm i @intuio/medusa-plugin-klaviyo

Follow the instruction in the readme for setup.

This package will be actively updated.
